src/tests/system/tests/test_gdm_smartcard.py

```diff
@@ -0,0 +1,128 @@
+from __future__ import annotations
```

Comment:
docstrings with the steps for the test cases are missing in this file and I don't see any assert in this file